jobs/furnishings/src/furnishings/stage_processors/post_processor.py

```diff
@@ -84,7 +84,7 @@ def process(self):
         self._format_furnishings()
         self._app.logger.debug('Formatted furnishing details presented in XML file')
         self._set_meta_info()
-        payload = self._build_xml_data(self._xml_data)
+        payload = self._build_xml_data(self._xml_data, self._processed_date.strftime('%I:%M %p'))
```

Comment:
If it's, let's say 15:05pm, this will output 03:05 PM.

Reply:
so in the screenshots you provided, the date doesn't contain the "0" before 3:14 PM. 

can you confirm there will be a "0" before 3:14 PM in your current code? 

![image](https://github.com/user-attachments/assets/d2b999db-729d-4cd6-bf37-f6ffd3fb880b)

Reply:
Oh yes, the screenshot was made before my 2nd commit. I changed that.

Reply:
It was basically the `-` that was before I. I removed it.

Reply:
Tested again now, looks good!
![new xml payload](https://github.com/user-attachments/assets/abfd4ff6-c9bf-4938-b4fe-28396ddbc4c1)